### PR TITLE
Update for Moodle 4.3 and PHP 8.2

### PR DESCRIPTION
**Pull request by justusdieckmann**
PHP 8.2 deprecates the creation of dynamic class properties.

### CODE REVIEW

File: vendor/elan-ev/opencast-api/src/OpencastApi/Opencast.php
Comment (ferishili) [inline vendor/elan-ev/opencast-api/src/OpencastApi/Opencast.php:1]: Hi @justusdieckmann,
Please keep in mind that any changes to external lib of Opencast PHP library must be handled and added into its repo, therefore, I would recommend providing these changes [here](https://github.com/elan-ev/opencast-php-library) and having that updated from the composer required command to keep everything under control.
Thanks in advance.

Comment (justusdieckmann) [reply vendor/elan-ev/opencast-api/src/OpencastApi/Opencast.php:1]: Hi @ferishili,
I even wanted to do that, but didn't thought of it just now. I have created the PR, can you create a new version for compose?
Thanks
Justus

Comment (ferishili) [reply vendor/elan-ev/opencast-api/src/OpencastApi/Opencast.php:1]: Thanks, I just saw your PR.
Of course, I will get that PR running and will release the new version.

Comment (ferishili) [reply vendor/elan-ev/opencast-api/src/OpencastApi/Opencast.php:1]: The release is out: v1.5.0
Thanks for your work

Comment (justusdieckmann) [reply vendor/elan-ev/opencast-api/src/OpencastApi/Opencast.php:1]: Thank you very much for the quick work!

**Comment by NinaHerrmann** [comment]
Thank both of you very much fr the fast changes! :) :partying_face:

**Pull request merged**